src/expressions/field-expr.md

```diff
@@ -4,28 +4,48 @@
 > _FieldExpression_ :\
 > &nbsp;&nbsp; [_Expression_] `.` [IDENTIFIER]
 
-A _field expression_ consists of an expression followed by a single dot and an [identifier], when not immediately followed by a parenthesized expression-list (the latter is always a [method call expression]).
+A *field expression* is a [place expression] that evaluates to the location of a field of a type.
+When the operand is [mutable], the field expression is also mutable.
+
+The syntax for a field expression is an operand, then a `.`, and finally an [identifier].
+Field expressions cannot be followed by a parenthetical comma-separated list of expressions, as that is parsed as a [method call expression].
+That is, they cannot be the function operand of a [call expression].
+
+> **Note**: Wrap the field expression in a [parenthesized expression] to use it in a call expression.
+>
+> ```rust
+> # struct HoldsCallable<F: Fn()> { callable: F }
+> let holds_callable = HoldsCallable { callable: || () };
+>
+> // Invalid: Parsed as calling the method "callable"
+> // holds_callable.callable();
+>
+> // Valid
+> (holds_callable.callable)();
+> ```
+
 A field expression denotes a field of a [struct] or [union].
+
 To call a function stored in a struct, parentheses are needed around the field expression.
 
 <!-- ignore: needs lots of support code -->
 ```rust,ignore
 mystruct.myfield;
 foo().x;
 (Struct {a: 10, b: 20}).a;
-mystruct.method();          // Method expression
 (mystruct.function_field)() // Call expression containing a field expression
 ```
 
-A field access is a [place expression] referring to the location of that field.
-When the subexpression is [mutable], the field expression is also mutable.
+## Automatic dereferencing
+
+Also, if the type of the operand implements [`Deref`] or [`DerefMut`][`Deref`] depending on whether the operand is [mutable], it is *automatically dereferenced* as many times as necessary to make the field access possible.
```

Comment:
Not sure what the "Also" is for.

```suggestion
If the type of the operand implements [`Deref`] or [`DerefMut`][`Deref`] depending on whether the operand is [mutable], it is *automatically dereferenced* as many times as necessary to make the field access possible.
```

Also, this removes the sentence about `In cases of ambiguity, we prefer fewer autoderefs to more.`.  Do you know what that was referring to?  What ambiguity is there?

Reply:
The ambiguity is if you can autoderef a type that has the field to another type that also has the field. The entire paragraph needs to be changed, but I'm trying to avoid making too many content changes in this PR, expecting other PRs for each expression to handle them.